tests/containers/agencyV2/accountSpending/TableContainer-test.jsx

```diff
@@ -67,15 +73,24 @@ test('no duplicate API Requests 👌👌👌👌 on fy change', () => {
     });
 });
 
-test('no duplicate API Requests 👌👌👌👌 on type change', () => {
+test('no duplicate API Requests on type change', () => {
     const spy = jest.spyOn(apis, 'fetchSpendingByCategory').mockReturnValue({
         promise: Promise.resolve({ data: mockResponse }),
         cancel: jest.fn()
     });
     const { rerender } = render(<TableContainer {...defaultProps} />);
     rerender(<TableContainer {...defaultProps} type="object_class" />);
     return waitFor(() => {
-        expect(spy).toHaveBeenCalledTimes(2);
-        expect(spy).not.toHaveBeenCalledTimes(3);
+        expect(spy).toHaveBeenCalledTimes(1);
+        expect(spy).not.toHaveBeenCalledTimes(2);
+    });
+});
+
+test('parses data from the API using agency obligated amount from redux state', () => {
```

Comment:
It looks like we are testing that `parseRows` is being called with data returned from the API.

I am wondering what value this test brings considering we are testing that the API (`fetchSpendingByCategory`) is being called, and after that API call the immediate execution of `parseRows` is called. When would `parseRows` not be called?